onnxruntime/test/providers/provider_test_utils.cc

```diff
@@ -194,7 +207,12 @@ void Check<BFloat16>(const OpTester::Data& expected_data, const Tensor& output_t
       EXPECT_EQ(f_expected[i], f_output[i]);
     else {
       // the default for existing tests
-      EXPECT_NEAR(f_expected[i], f_output[i], threshold) << "provider_type: " << provider_type;
+      const float max_value = fmax(fabs(f_expected[i]), fabs(f_output[i]));
+      if (max_value != 0)  // max_value = 0 means output and expected are 0s.
+      {
+        const float rel_error = fabs(f_expected[i] - f_output[i]) / max_value;
```

Comment:
nit: please run clang-format so the formatting conforms to the coding standard

Reply:
Brace layout is still off so doesn't look like clang-format ran correctly.

Reply:
hmmm.. ok will check, thx